README.md

```diff
@@ -73,35 +86,35 @@ module "lambda_api" {
 * DNS A-Record
 
 ## Requirements
-* Terraform version 0.12.21 or greater
-* AWS provider version 2.56 or greater
+* Terraform version 0.13.2 or greater
+* AWS provider version 3.0 or greater
 
 ## Inputs
 | Name | Type  | Description | Default |
 | --- | --- | --- | --- |
-| app_name | string | application name |
-| env | string | application environment (e.g. dev, stg, prd) |
+| app_name | string | application name (include the env aka. 'my-api-dev') | |
```

Comment:
might be useful to mention the 28 character limit here

Reply:
Good idea. I copy-pasted from fargate-api module and even added a conditional to the variable itself